products/cdk/get-started-cdk2-ruby.adoc

```diff
@@ -64,7 +64,7 @@ include::products/cdk/_includes/get-started-cdk2-step2b.adoc[]
 
 In this step, you will create a tiny Hello World container that uses {tthw-langshort} as a web server. Once created, the container can be run on other systems that have `docker` installed.  You will need to create several files in an empty directory using your favorite editor, including a `Dockerfile` that describes how to build the container image.
 
-Note: you can edit files on your host system that can be synchronized to your vagrant box with `vagrant rsync`. For more information see _Vagrant synchronizad folders_ in the _CDK Installation Guide_. 
+Note: When you use SSHFS, your home directory from the host system is automatically synchronized to your Vagrant box. For more information, see https://access.redhat.com/documentation/en/red-hat-container-development-kit/2.1/getting-started-guide/#synced-folders[Vagrant Synchronizad Folders] in the _CDK Getting Started Guide_. 
```

Comment:
FYI, There are 4 files here with the same wording covering Node, PHP, Python, and Ruby to keep in sync.  This commit hits only two. 

With this edit SSHFS seems a little out of context.  Suggest something like:

You can edit files on your host system and have them automatically synchronized to your Vagrant box using SSHFS. See https://access.redhat.com/documentation/en/red-hat-container-development-kit/2.1/getting-started-guide/#synced-folders[Vagrant Synchronizad Folders] in the _CDK Getting Started Guide_.